### PR TITLE
htop: update pt_BR translation

### PR DESCRIPTION
**Pull request by Guilospanck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.pt_BR/common/htop.md [hunk 1]
```diff
@@ -9,8 +9,20 @@
 
 - Inicializar htop mostrando somente processos pertencentes a um usuário:
 
-`htop -u {{nome_usuário}}`
+`htop --user {{nome_usuário}}`
 
-- Obter ajuda acerca de comandos interativos:
+- Ordenar processos por um `item-de-ordenacao` (utilize `htop --sort help` para ver as opçōes disponíveis):
```
Comment (navarroaxel) [inline pages.pt_BR/common/htop.md:14]:
```suggestion
- Ordena processos por um `item-de-ordenacao` (utilize `htop --sort help` para ver as opçōes disponíveis):
```

File: pages.pt_BR/common/htop.md [hunk 2]
```diff
@@ -9,8 +9,20 @@
...
+
+`htop --sort {{item-de-ordenacao}}`
+
+- Ver comandos interativos enquanto roda `htop`:
```
Comment (navarroaxel) [inline pages.pt_BR/common/htop.md:18]:
```suggestion
- Vê comandos interativos enquanto roda `htop`:
```

File: pages.pt_BR/common/htop.md [hunk 3]
```diff
@@ -9,8 +9,20 @@
...
 
 `?`
+
+- Mudar para uma aba diferente:
+
+`tab`
+
+- Mostrar ajuda:
```
Comment (navarroaxel) [inline pages.pt_BR/common/htop.md:26]:
```suggestion
- Mostra ajuda:
```

File: pages.pt_BR/common/htop.md [hunk 4]
```diff
@@ -9,8 +9,20 @@
...
-- Obter ajuda acerca de comandos interativos:
+- Ordenar processos por um `item-de-ordenacao` (utilize `htop --sort help` para ver as opçōes disponíveis):
+
+`htop --sort {{item-de-ordenacao}}`
+
+- Ver comandos interativos enquanto roda `htop`:
 
 `?`
+
+- Mudar para uma aba diferente:
```
Comment (navarroaxel) [inline pages.pt_BR/common/htop.md:22]:
```suggestion
- Muda para uma aba diferente:
```

File: pages.pt_BR/common/htop.md [hunk 5]
```diff
@@ -9,8 +9,20 @@
 
 - Inicializar htop mostrando somente processos pertencentes a um usuário:
```
Comment (navarroaxel) [inline pages.pt_BR/common/htop.md:10]:
```suggestion
- Inicializa htop mostrando somente processos pertencentes a um usuário:
```

File: pages.pt_BR/common/htop.md [hunk 6]
```diff
@@ -7,10 +7,22 @@
...
-- Inicializar htop mostrando somente processos pertencentes a um usuário:
+- Inicializa htop mostrando somente processos pertencentes a um usuário:
 
-`htop -u {{nome_usuário}}`
+`htop --user {{nome_usuário}}`
 
-- Obter ajuda acerca de comandos interativos:
+- Ordena processos por um `item-de-ordenacao` (utilize `htop --sort help` para ver as opçōes disponíveis):
+
+`htop --sort {{item-de-ordenacao}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/htop.md:16]: We use `snake_case` for placeholders
```suggestion
`htop --sort {{item_de_ordenacao}}`
```

File: pages.pt_BR/common/htop.md [hunk 7]
```diff
@@ -7,10 +7,22 @@
...
 `htop`
 
-- Inicializar htop mostrando somente processos pertencentes a um usuário:
+- Inicializa htop mostrando somente processos pertencentes a um usuário:
 
-`htop -u {{nome_usuário}}`
+`htop --user {{nome_usuário}}`
 
-- Obter ajuda acerca de comandos interativos:
+- Ordena processos por um `item-de-ordenacao` (utilize `htop --sort help` para ver as opçōes disponíveis):
```
Comment (navarroaxel) [inline pages.pt_BR/common/htop.md:14]:
```suggestion
- Ordena processos por um `item_de_ordenacao` (utilize `htop --sort help` para ver as opçōes disponíveis):
```

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
@kbdharun please

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**